cloud/services/managedclusters/service.go

```diff
@@ -0,0 +1,33 @@
+/*
+Copyright 2020 The Kubernetes Authors.
```

Comment:
should this be in https://github.com/kubernetes-sigs/cluster-api-provider-azure/blob/master/exp/cloud/services ?

Reply:
I was a little surprised we put the services in exp/, I can move this to match though. There was some funkiness with credentials in scope but I think I just need to make some things public.

Reply:
yeah, same thing as above. I'm okay with either but we should be consistent.

Reply:
I vote for moving the cloud stuff out of `./exp`.

Reply:
I'm happy to make the change in a follow up PR.

Reply:
Cool so let's put it all in /exp in this PR then since there's already some of it in exp/ in current master. Unless you want to make a prequel PR @devigned and rebase this one on top of it.

Reply:
I'm moving it right meow :cat:

Reply:
@devigned wait can we decide who is doing it? I already started halfway and then flipped back after seeing Cecile's comment. I'll keep in in exp in this PR unless you say otherwise?

Reply:
guess that's an answer 😋

Reply:
I'm good either way. It's a small PR. I'm happy to rebase after this gets pulled in, or have it merged and you rebase. I think the former is easier than the latter.

Reply:
🚢 it!